
\section{Methods}

\subsection{Chromosome data}
We downloaded all available chromosome data for the clade Polyneoptera from the Tree of Sex database \citep{blackmon2016,TOS2014}.
However, some species were present with only limited data (e.g., species that had a sex chromosome system listed but no chromosome number). 
For these species we performed focused literature searches for species that were listed in the database but had incomplete records.
This yielded a final data set of 783 records. 

To maximize the overlap between our karyotype and sequence data sets (discussed below) we first found all species level exact matches in both data sets.
Next, we looked for genera level matches that lacked species level matches.
For each of these matches we retained one randomly chosen species on the phylogeny to act as a exemplar for the clade and renamed it as such. 
When there were multiple karyotype records for a genus or species all were retained.
In our data set there were 25 species with multiple chromosome numbers reported in the literature.
Furthermore, our process of maximizing overlap between the two data sets created 57 exemplar taxa (genera tips) with multiple chromosome numbers.

\subsection{Phylogenetic data}
We used PyPHLAWD to retrieve sequence clusters and used clusters which had at least 100 species for our analysis \citep{smith2018phyphlawd}. 
These included three mitochondrial genes (COI, COX2 and ND4) and three nuclear regions (18s and two regions of the 28s gene). 
We removed duplicate sequences and retained the longest example for each species using the function FastaFilter in the R package evobiR \citep{blackmon2013EvoBiR}.
We used MAFFT under default settings to align all sequences \citep{katoh2013mafft}.
For the aligned RNA coding sequences, we used GBLOCKS to remove hyper variable regions \citep{castresana2000gblocks}. 
When running GBLOCKS we used default settings with the exception of the allowed gap positions argument which was set to maximum. 
For 18S sequence cluster we also set the minimum block length to 6 to retain a greater proportion of the alignment. 
For the protein coding genes, We manually adjusted starting position of the alignments to maintain the reading frame. 
Using the supermatrix function in the R package evobiR individual gene alignments were then concatenated into a supermatrix with 7380 sites \citep{blackmon2013EvoBiR}.
When there were more than one unit in the sequence data set for genus level matches, and when there were more than one sequence for a particular gene or region, we retained the longest sequence upon concatenation. If sequences were of equal length then we picked a sequence randomly.

The presence of rouge taxa (taxa that have inconsistent placement in a set of phylogenetic trees) can produce unreliable rate inferences similar to that found in analyses of supertrees in \cite{rabosky2015b}.
To identify the presence of rouge taxa, we generated 100 maximum likelihood rapid bootstrap trees using RAxML v 8.2.10 under CIPRES Science Gateway \citep{stamatakis2014raxml,miller2010cipres}.
Using these trees we calculated the taxonomic instability index as implemented in Mesquite v 3.51 \citep{maddison2018mesquite}.
When we examined taxonomic instability indices we found that a score of 4870 was an inflection point (\cref{fig:tax.index}) \citep{aberer2012roguetaxa}.
We identified 16 taxa whose taxonomic instability index was higher that this inflation point and removed them from subsequent analysis.
Our final dataset contained 232 taxonomic units with 72.729\% missing data.

We used BEAST version 2.5 \citep{bouckaert2014beast} to infer time calibrated phylogenies under a relaxed log normal clock and using birth-death model as the speciation model and GTR + G as the nucleotide substitution model.
A total of six calibration points were used that were taken from a previous study to time calibrate the phylogenetic tree \citep{misof2014phylogenomics}.
For each of these calibration points we used a normal distribution.
The upper and lower bounds of the calibration points (95\textsuperscript{th} and 5\textsuperscript{th} percentiles respectively) were placed according to the confidence intervals as presented in \citet{misof2014phylogenomics}. 
We conducted two independent runs, each for 100 million generations.
Convergence of these two independent runs was evaluated using Tracer v 1.7 \citep{rambaut2018tracer}.
Initial 50\% of each run was discarded as burn-in and 50 phylogenetic trees were randomly sampled from the post-burnin period of each run to construct a posterior distribution of 100 trees used for trait analyses described below.

\subsection{Modeling chromosome evolution}
We used R packages diversitree \citep{fitzjohn2012} and chromePlus \citep{blackmon2019meiotic} to model the chromosome number evolution in a Bayesian framework.
To get reliable estimates for the rates of chromosome number evolution we only evaluated orders with at least 20 taxonomic units which resulted in four orders.
We tested two versions of our model, a simpler model with chromosome gains (fission) and losses (fusion) as the parameters and a complex model which included fusion, fission and polyploidy as the parameters.
Based on the likelihood ratio test results we used the complex model to estimates the rates of chromosome changes at the order level.

We conducted an MCMC run of 1000 generations for each order on each of the 100 trees.
We tested whether there are differences in the rates of chromosome changes at the Order level.
Inspection of the parameter estimates revealed that our MCMC runs converged by 50 generations.  
We discarded the initial 25\% as burn-in and randomly sampled 100 states from the remaining run. 

\subsection{Ancestral state reconstruction}
We estimated the ancestral state of the root of the four orders using ChromEvol version 2.0. \citep{glick2014chromevol, mayrose2009chromevol}.
we used a fixed parameter model as the chromosome number evolutionary model which included three parameters of chromosome evolution.
They are 1) chromosome gains, 2) chromosome losses and 3) whole genome duplication.
As the phylogenetic input we used the 100 trees sampled from the initial phylogenetic reconstruction. 
For each tree, we took the average of the model parameters from the corresponding post burnin portion of the ChromPlus model and used these values as the parameter values in the ancestral state reconstruction.
The chromosome counts for each order were taken from the Tree of Sex database \citep{blackmon2016,TOS2014}.
When there were multiple chromosome counts reported for a taxonomic unit, we randomly sampled a single chromosome number and used it as the exemplar chromosome count for that particular taxonomic unit.

The ancestral state for the sex chromosome system of the Polyneoptera clade was reconstructed using the R package APE \citep{Paradis2018}.
We classified multi-XY sex chromosome system as XY which resulted in two states (XO and XY) for the ancestral states reconstruction of the sex chromosome system.
A markov model was used where transitions between X0 and XY allowed to occur unequal rates.

R code for all analyses are provided in the supplementary file 1.