\section{Results}

\subsection{Evolution of sex chromosome systems}
In our data set 21 genera contain species with different types of SCSs (i.e. XO XY, and or multi-XY).
In each of these genera we calculate the mean number of chromosome for all species with a given type of SCS.
By comparing these means within genera we can determine if differences are consistent with fusions or fissions as a source of transitions among SCSs.
Briefly, if transitions from XO to XY are generated by the fusion of an autosome to a sex chromosome we would expect a lower mean chromosome number for XY species.
Likewise, if transitions from XY to multi-XY are generated by the fusion of an autosome to a sex chromosome we would expect a lower mean chromosome number for multi-XY species.
In contrast, if transitions from XY to multi-XY are generated by the fission of a sex chromosome we would expect a higher mean chromosome number for multi-XY species.

We find strong support for fusions as a source of transitions from XO to XY SCSs.
Of the 16 genera with both XO and XY species 94\% (15/16) show a lower mean chromosome number in XY species (\cref{tab:fusions}). 
However, we find little support for fusions leading to transitions from XY to multi-XY.
In fact, only one of the seven genera with both XY and multi-XY has a lower mean chromosome number for multi-XY species.
Instead, 71\% (5/7) of the genera with both XY and multi-XY have a higher mean chromosome number in multi-XY species.
This analysis is limited to only those genera with variation in SCSs and thus omits much our data.
Examining the distribution of chromosome number among all species in each order parsed by SCS suggests that in some groups the origin of transitions may differ.
For instance, the mean of all XY species in both Blattodea and Dermaptera is higher than the mean of XO species in both orders.
This pattern is not expected if fusions are the primary source of transitions from XO to XY in these groups (\cref{fig:order.plots}).

\subsubsection{Ancestral states and rates of sex chromosome evolution}

We find that the ancestral state for SCS in Polyneoptera clade was XO with a probability of 90.3\%.
Similarly, the most probable ancestral state for of each order was also XO, with the exception of Isoptera and Dermaptera where XY is more probable.
(\cref{fig:sex.asr.plot}).
We find that the credible interval of the transition rates from XO to XY and XY to XO to be largely overlapping with means of 0.00202 and 0.00200 respectively. 
To assess the number of transitions between XY and XO we calculated the number of transitions from 100 stochastic maps for each of the 100 trees from the posterior sample.
Transitions from XO to XY were more common (mean = 15.3) while transitions from XY to XO were relatively rare (mean = 6.7).

\subsection{Chromosome number variation}
We find a significant difference in variance in chromosome numbers among orders of Polyneoptera (Levene's test p-value : 2.2e-16). 
Blattodea, having 172 records, had the highest variance in chromosome number (39.32) and  Embiidina, with eight records, had the lowest variance in chromosome number (0.41).
Orthoptera, despite having 284 records, had a low variance in chromosome number (1.39). 
Posthoc tests show that Blattodea has higher variance in chromosome number than Embidiina, Mantodea, Orthoptera, Phasmatodea, Plecoptera (all p-values < 0.05). 
Likewise Dermaptera has higher variance in chromosome number than Mantodea and Orthoptera (all p-values < 0.05). 
Finally, Orthoptera has lower variance in chromosome number than Phasmatodea (p-value < 0.05).
Much of these differences in variance are obvious even when looking at the reduced phylogenetic dataset in \cref{fig:phyloplot}.

\subsection{Chromosome Number Evolution Rate inference}
Though differences in chromosome number variance suggest some orders are evolving more quickly we must control for the phylogeny to draw any rigorous conclusions.
We began with a base model that allows for fusions and fissions and compared this via a likelihood ratio test with a model that included polyploidy as well.
We repeated this likelihood ratio test on each of the 100 trees from our posterior distribution for each order.
All orders showed some support for the model including polyploidy and overall 77.6\% of our likelihood ratio tests supported an important role for polyploidy.
For this reason all analyses were done with a model that had fusions, fissions, and polyploidy.
In Blattodea we estimate a mean fusion rate of 0.128, a fission rate of 0.150, and a polyploidy rate of 0.003 (\cref{tab:HPD}).
In contrast if we remove the subclade Isoptera from Blattodea we find that parameter estimates increase to 0.420, 0.385, and 0.004 for fusions, fissions and polyploidy respectively.
This is consistent with rate estimates for Isoptera in isolation, where we infer a mean fusion rate of 0.044, a fission rate of 0.063 and a polyploidy rate of 0.003.
In Mantodea we estimate a mean fusion rate of 0.142, a fission rate of 0.056, and a polyploidy rate of 0.139.
Mantodea rate estimates also exhibited high uncertainty overlapping rate estimates in most orders for most parameters (\cref{fig:rates}).
In Phasmatodea we estimate a mean fusion rate of 0.47, a fission rate of 0.145, and a polyploidy rate of 0.039.
Some of the lowest rates we estimated were in Orthoptera where the fusion rate was 0.003 and the fission rate was 0.024. However we do find the polyploidy rate to be high with a mean rate of 0.101.

Genome size data was available for 123 Polyneoptera species in the orders Phasmatodea, Orthoptera, Mantodea, and Blattodea (including Isoptera).
Mean genome size of these orders ranged from a low 1.23 Gbp in Isoptera to a high of 7.92 Gbp in Orthoptera (\cref{tab:genomesizes}).
We fit three linear models where the mean rate of chromosome evolution (gains, losses, and polyploidy) were a function of the mean genome size of an order.
None of these linear models was significant (all p-values > 0.15 without correction for multiple comparisons).
Furthermore, some of the very lowest rates were observed in Orthoptera which had the largest mean genome size.

\subsection{Chromosome Number Ancestral state reconstruction}
In reporting ancestral chromosome number in each order we averaged probabilities across all 100 trees from the posterior distribution. 
Our analysis of Orthoptera suggests six is the most probable ancestral state for the order, with a 75.1\% probability.
However, a chromosome number of three is given a probability of 24.5\% which may suggest an early whole genome duplication in this clade.
Support for the lower ancestral state was concentrated in 19 trees where the most probable state was 3 in all other trees 6 was given the highest probability (\cref{fig:asr}). 
In Blattodea, the most probable chromosome number was 13 with a probability of 20.9\% followed by 14 and 12 with probabilities 19.5\% and 16.4\% respectively. 
In Blattodea without Isoptera, the most probable chromosome number was 7 with a probability of 11.9\% followed by 8 and 9 with probabilities 11.6\% and 11.2\% respectively.
In Isoptera, the most probable chromosome number was 20 with a probability of 35.6\% followed by 21 and 22 with probabilities 30.7\% and 14.0\% respectively.
In Mantodea, the most probable chromosome number was 8 with a probability of 41.2\% followed by 9 and 7 with probabilities 19.2\% and 16.1\% respectively.
Finally, in Phasmatodea, the most probable chromosome number was 9 with a probability of 11.7\% followed by 10 and 11 with probabilities 11.3\% and 10.7\% respectively.

\subsection{Asexuality and rates of chromosome evolution}
In our dataset, we find 13 species which are parthenogenetic.
All these species were within Phasmatodea.
Using our expanded Phasmatodea only phylogenies we tested whether the rates of chromosome evolution are contingent on the reproduction mode (\cref{fig:phas.phylo}).
We found that there is no significant difference in the rates chromosome fusion and fission between sexually and asexually reproducing species.
However, we find that rates of polyploidy is significantly higher in asexually reproducing species than in sexually reproducing species (\cref{fig:phas.plot}).
